src/js/components/modals/AppModalComponent.jsx

```diff
@@ -115,13 +106,9 @@ var AppModalComponent = React.createClass({
   },
 
   onInfoChange: function () {
-    this.removeInfoListeners();
-
-    if (!this.hasInfoObject()) {
-      this.setState({
-        info: InfoStore.info
-      });
-    }
+    this.setState({
```

Comment:
You setting the state now, even is there is a request error at the info endpoint.
Is this intentional?